### PR TITLE
v6.9.0

### PR DESCRIPTION
**Pull request by LukasTy**
### Release the packages

- [ ] Checkout the last version of the working branch
- [ ] Make sure you have the latest dependencies installed: `yarn`.
- [ ] Build the packages: `yarn release:build`.
- [ ] Release the versions on npm: `yarn release:publish` (you need your 2FA device).
- [ ] Push the newly created tag: `yarn release:tag`.

### Publish the documentation

The documentation must be updated on the `docs-vX` branch (`docs-v4` for `v4.X` releases, `docs-v5` for `v5.X` releases, ...)

- [ ] Push the working branch on the documentation release branch to deploy the documentation with the latest changes.

```sh
git push upstream master:docs-v6 -f
```

You can follow the deployment process [on the Netlify Dashboard](https://app.netlify.com/sites/material-ui-x/deploys?filter=docs-v6)
Once deployed, it will be accessible at https://material-ui-x.netlify.app/ for the `docs-v6` deployment.

### Publish GitHub release

- [ ] After documentation is deployed, publish a new release on [GitHub releases page](https://github.com/mui/mui-x/releases)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,76 @@
...
+
+- 🎁 We release a new package `@mui/x-charts`.
+  This package aims at simplifying the integration of charts into your dashboards. 📊
+
+  ![image](https://github.com/mui/mui-x/assets/45398769/613bae2f-537d-4ec3-8046-591d1a504a72)
+
+  It already contains [lines](https://mui.com/x/react-charts/lines/), [bars](https://mui.com/x/react-charts/bars/), and [scatter](https://mui.com/x/react-charts/scatter/) charts with basic customization features.
+  Please, visit the new [documentation](https://mui.com/x/react-charts/) to know what it does, and open issues to get the feature you need to be implemented.
+
+- 🚀 Introduce the UTC and timezone support for pickers.
```
Comment (LukasTy) [inline CHANGELOG.md:20]: @flaviendelangle Feel free to suggest a better highlight 🙈

File: packages/x-codemod/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-codemod",
-  "version": "6.2.1",
+  "version": "6.9.0",
```
Comment (LukasTy) [inline packages/x-codemod/package.json:3]: I'm thinking about syncing the version just for the sake of aligning the Babel versions.
What do you think?

File: packages/x-license-pro/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-license-pro",
-  "version": "6.6.0",
+  "version": "6.9.0",
```
Comment (LukasTy) [inline packages/x-license-pro/package.json:3]: We've missed releasing it last week due to https://github.com/mui/mui-x/pull/9135 🙈

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,76 @@
...
+- [DataGrid] Filtering performance: use unmemoized selectors by default (#9287) @romgrk
+- [DataGrid] Use container dimensions from `getComputedStyle` (#9236) @m4theushw
+- [l10n] Improve Brazilian Portuguese (pt-BR) locale (#9404) @julioAz
+
+### `@mui/x-date-pickers@v6.9.0` / `@mui/x-date-pickers-pro@v6.9.0`
+
+#### Changes
+
+- [fields] Ensure `minutesStep` is respected by fields arrows up/down (#9338) @alexfauquette
+- [fields] Reset internal state when referenceValue changes (#9390) @adrianmxb
```
Comment (flaviendelangle) [inline CHANGELOG.md:42]:
```suggestion
- [fields] Reset internal state when `referenceValue` changes (#9390) @adrianmxb
```

File: CHANGELOG.md (hunk 1 above)
Comment (flaviendelangle) [reply CHANGELOG.md:20]: Maybe we can add a small GIF, something like:

ezgif-2-92aed425de

**Review by alexfauquette — APPROVED** [review]
🚀🚀🚀

**Review by flaviendelangle — APPROVED** [review]
:rocket: That's a big release :star_struck:

**Comment by oliviertassinari** [comment]
I have set https://github.com/mui/mui-x/releases/tag/v6.9.0 as the latest

Screenshot 2023-06-23 at 19 01 01

It was displaying v6.8.0 as latest.

**Comment by LukasTy** [comment]
Thank you @oliviertassinari, my bad. 🤦 🙈

**Pull request merged**